src/core/lib/security/authorization/rbac_translator.cc

```diff
@@ -398,16 +395,17 @@ absl::StatusOr<RbacPolicies> GenerateRbacPolicies(
             allow_policy_or.status().code(),
             absl::StrCat("allow_", allow_policy_or.status().message()));
       }
-      allow_rbac = absl::make_unique<Rbac>(std::move(allow_policy_or.value()));
+      allow_rbac = std::move(allow_policy_or.value());
```

Comment:
Same as above: this can say `*allow_policy_or` instead of `allow_policy_or.value()`, since you've already checked that it's not an error above.

Reply:
Done